pep8.py

```diff
@@ -1890,7 +1898,14 @@ def excluded(self, filename, parent=None):
 
         Check if 'options.exclude' contains a pattern that matches filename.
         """
-        if not self.options.exclude:
+        result = self._excluded(filename, parent=parent)
+        if result and self.options.verbose:
+            print('ignoring excluded file: %s' % filename)
+        return result
+
+    def _excluded(self, filename, parent=None):
+        patterns = self.options.exclude
```

Comment:
Should probably update the other references in the function to `self.options.exclude` to use this `patterns` variable.